tests/integration/targets/pipx/tasks/main.yml

```diff
@@ -339,3 +339,36 @@
   assert:
     that:
       - install_pyinstaller is changed
+
+##############################################################################
+# Test for issue 8656
+- name: ensure application conan2 is uninstalled
+  community.general.pipx:
+    name: conan2
+    state: absent
+
+- name: Install Python Package conan with suffix 2 (conan2)
+  community.general.pipx:
+    name: conan
+    state: install
+    suffix: "2"
+  register: install_conan2
+
+- name: Install Python Package conan with suffix 2 (conan2) again
+  community.general.pipx:
+    name: conan
+    state: install
+    suffix: "2"
+  register: install_conan2_again
```

Comment:
How about also installing `conan`, and checking that this doesn't change the behavior for installing and removing `conan2`?

Reply:
Doable of course, but IMO that would prove that pipx - -suffix (command) works, not that pipx (module) does or does not.

Reply:
I guess it's both. Such a test would also catch if the module simply ignores the `suffix` parameter completely.

Reply:
Happy to add it (later on) but again, if the module ignored `suffix` the second assertion below would fail. We're just proving the *command* does what it claims.

Reply:
Fair point.